Common/Product/SharedProject/Wpf/Controls.xaml

```diff
@@ -335,7 +335,8 @@
                 <ControlTemplate TargetType="{x:Type Button}">
                     <ContentPresenter TextElement.Foreground="{TemplateBinding Foreground}" RecognizesAccessKey="True" Height="Auto">
                         <ContentPresenter.Resources>
```

Comment:
Did you test with all themes? Also, there's a few copy of this file that probably should stay in sync.

Reply:
I'll do both of those. Shouldn't affect the themes as it's just making explicit the previously implicit values (which were broken by injecting more styles into the mix...)